betterstack.com.status-page.json

```diff
@@ -0,0 +1,19 @@
+{
+    "providerId": "betterstack.com",
+    "providerName": "Better Stack",
+    "serviceId": "status-page",
+    "serviceName": "Status page",
+    "version": 1,
+    "logoUrl": "https://betterstack.com/static/images/better-stack-logo.png",
+    "description": "Configures a sub-domain for use as a Better Stack status page",
+    "syncPubKeyDomain": "betterstack.com",
+    "hostRequired": true,
+    "records": [
+        {
+            "type": "CNAME",
+            "host": "@",
+            "pointsTo": "statuspage.betteruptime.com",
+            "ttl": 60
```

Comment:
Why would you want so short TTL?

Reply:
That's the default we use in our internal systems so I used it here too, but you're right, it doesn't make sense for the users as they will pretty much never change this record. I'll change it to a logner value.